### DISCUSSION TITLE
crazyswarm import error after Installation

### DISCUSSION THREAD

**Discussion by CookiePPP** [USC-ACTLab/crazyswarm#680] [Q&A] [answered]
I followed the [Installation Guide](https://crazyswarm.readthedocs.io/en/latest/installation.html) for Physical Robots and Simulation.

However I am stuck at the [Hello World Tutorial](https://crazyswarm.readthedocs.io/en/latest/tutorials/tutorials.html).
I get the below error when running `helloworld.py` without `--sim` argument.

```
blimpy@blimpy-OMEN:~/Documents/crazyswarm/ros_ws/src/crazyswarm/scripts$ python3 hello_world.py
Traceback (most recent call last):
  File "hello_world.py", line 22, in <module>
    main()
  File "hello_world.py", line 11, in main
    swarm = Crazyswarm()
  File "/home/blimpy/Documents/crazyswarm/ros_ws/src/crazyswarm/scripts/pycrazyswarm/crazyswarm_py.py", line 50, in __init__
    from .crazyflie import TimeHelper, CrazyflieServer
  File "/home/blimpy/Documents/crazyswarm/ros_ws/src/crazyswarm/scripts/pycrazyswarm/crazyflie.py", line 12, in <module>
    from crazyswarm.srv import *
ModuleNotFoundError: No module named 'crazyswarm'
```
I've checked pip and crazyswarm package is not installed. I followed the installation instructions a second time and was more careful to read every output and ensure all commands ran successfully but I didn't see anything that suggested a command failed.

Can someone help me install the package locally?
I can see the relavent files in `crazyswarm/ros_ws/devel/lib/python3/dist-packages/crazyswarm/srv` but don't know how to install them to pip.

PS:
The self-test passes and both `chooser.py` and the crazyswarm_server run successfully.

**Comment by uhvardhan** [comment, marked as answer]
In the crazyswarm folder, can you type the following `source ros_ws/devel/setup.bash`? 

Once you run the above command, you should be able to run `python3 hello_world.py`

**Reply by CookiePPP** [reply]
`source ros_ws/devel/setup.bash` doesn't appear to do anything and doesn't print anything in terminal.

I have to wait till tomorrow to access the motion capture area. If it's alright with you, I'd like to wait for tomorrow to continue debugging this so I can be more responsive.

**Reply by uhvardhan** [reply]
When you run `source ros_ws/devel/setup.bash`, it won't print anything. But then you should be able run `hello_world.py` without issues.

**Reply by CookiePPP** [reply]
@uhvardhan Thanks, everything ran today without issue.
I'm really confused why it didn't work the first time, but I must've made a mistake somewhere on my first attempt(s).
